products/memgraph/v0.51.0/reference_guide/query-modules.md

```diff
@@ -6,33 +6,8 @@ startup.
 
 ### Loading Query Modules
 
-The Memgraph installation comes with the `example.so` and `py_example.py` query
-modules which are located in `/usr/lib/memgraph/query_modules` directory,
-Assuming the standard installation on Debian, you would run Memgraph with the
-following command.
-
-```plaintext
-systemctl start memgraph
-```
-
-When using Docker, the equivalent would be the following.
-
-```plaintext
-docker run -p 7687:7687 \
-  -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph \
-  memgraph
-```
-
 Memgraph will now attempt to load the query modules form all `*.so` and `*.py`
```

Comment:
The `now` word seems odd here. Now after what?
Maybe write something like `Upon startup, Memgraph will attempt...`